qa/tasks/cephfs/test_volumes.py

```diff
@@ -607,6 +613,42 @@ def test_volume_rename_for_more_than_one_data_pool(self):
         # data pool names unchanged
         self.assertCountEqual(orig_data_pool_names, list(self.fs.data_pools.values()))
 
+    def test_volume_info(self):
+        """Tests the 'fs volume info' command
```

Comment:
nit: `Tests the 'fs volume info' command` on the next line?

Reply:
Done